pkg/scanners/kubernetes/scanner_test.go

```diff
@@ -614,3 +614,120 @@ spec:
 	assert.Equal(t, 14, firstResult.Metadata().Range().GetEndLine())
 	assert.Equal(t, "k8s.yaml", firstResult.Metadata().Range().GetFilename())
 }
+
+// TODO(simar): Uncomment once all k8s policies have subtype selector added
```

Comment:
Note: I've left this in as "failing example" that should "pass" when we add selectors to all Kubernetes policies.

Reply:
cc @chen-keinan just a heads up.